core/mondoo-aws-security.mql.yaml

```diff
@@ -2065,11 +2149,14 @@ queries:
         title: Terraform Documentation - AWS Provider
   - uid: mondoo-aws-security-ec2-volume-inuse-check
     title: Ensure EBS volumes attached to EC2 instances are configured for deletion on instance termination
-    impact: 30
-    filters: |
-      asset.platform == "aws-ec2-volume"
```

Comment:
This check was also originally on ` asset.platform == "aws-ec2-volume"`, so we should keep the single instance check below, too.